app/elements/challenge/challenge-modal.html

```diff
@@ -69,7 +70,17 @@
 			onChallengeResponse: function (res) {
 				this.fire('challengeCreated');
 				page.redirect('/challenges/' + res.detail.response.Key);
+			},
+			_computeDuration: function(){
+				var duration = 0;
+				for (var i = 0; i < this.selectedTasks.length; i++){
+					duration += this.selectedTasks[i].Duration;
+				}
+				this.Duration = duration / 6 / 1e10;
```

Comment:
OK, was not clear to me from only looking at the code. A comment on *why* you do stuff sometimes helps.

Please file an issue on what you were saying in the second paragraph.